mint/run/core/aws-sdk-go/quick-tests.go

```diff
@@ -609,6 +609,114 @@ func testObjectTagging(s3Client *s3.S3) {
 	}
 }
 
+// Tests bucket re-create errors.
+func testCreateBucketError(s3Client *s3.S3) {
+	// initialize logging params
+	startTime := time.Now()
+	function := "testMakeBucketError"
+	bucketName := randString(60, rand.NewSource(time.Now().UnixNano()), "aws-sdk-go-test-")
+	args := map[string]interface{}{
+		"bucketName": bucketName,
+	}
+	_, err := s3Client.CreateBucket(&s3.CreateBucketInput{
+		Bucket: aws.String(bucketName),
+	})
+	if err != nil {
+		failureLog(function, args, startTime, "", "AWS SDK Go CreateBucket Failed", err).Fatal()
+		return
+	}
+	defer cleanup(s3Client, bucketName, "", function, args, startTime, true)
+
+	_, errCreating := s3Client.CreateBucket(&s3.CreateBucketInput{
+		Bucket: aws.String(bucketName),
+	})
```

Comment:
Not sure, why multiple `CreateBucket` calls here?

Reply:
@nitisht this is for check the error in creating bucket if this bucket already exists